cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -522,6 +522,9 @@ $(document).ready(function () {
 			var dropdown = nav.find('.cms_submenu-dropdown');
 			var offset = parseInt(dropdown.data('top'));
 
+			//fix issue #4407
+			nav.parents('.cms_draggable').data( 'menustate', 1).trigger('cms.structureboard.menustate.change');
```

Comment:
would probably be simpler if `true` or `false` was used as a value, that would eliminate the need for `> 0` checks